### ISSUE TITLE
Training data of Yolo v1

### ISSUE THREAD

**Issue by meet-minimalist** [parlstrand/ml_playground#1]
Can you share loss decrease during in Yolo v1 during training? I am strugling a lot to achieve good performance. I am overfitting a lot.
Also provide final mAP of the Yolo v1.

**Comment by MrParosk** [comment]
I have not tested it on a validation set since I was simply interested in implementing the architecture; try to add some regularization to the model and see if it still overfits.
I will try to update the models in the coming weeks when I have time.
